requirements.txt

```diff
@@ -9,7 +9,9 @@ brotlipy
 pyyaml
 werkzeug
 webencodings
-gevent==1.2.2
+dnspython     # for updated gevent: recommend by gevent, c-ares resolver to be deprecated soon
+idna          # for updated gevent: Python 2.7 support
+gevent==1.3.3 # update requirement. Linux needs manual source install
```

Comment:
Seems like 1.3.4 for is out and has some wsgi related optimizations (http://www.gevent.org/changelog.html#id1) so should upgrade to it instead.

Reply:
done